dart/lib/src/protocol/sentry_event.dart

```diff
@@ -88,23 +92,25 @@ class SentryEvent {
   ///
   /// It's `runtimeType` and `toString()` are logged.
   /// If it's an Error, with a stackTrace, the stackTrace is logged.
-  /// If this behavior is undesirable, consider using a custom formatted [message] instead.
+  /// If this behavior is undesirable, consider using a custom formatted
+  /// [message] instead.
   dynamic get throwable => (_throwable is ThrowableMechanism)
       ? (_throwable as ThrowableMechanism).throwable
       : _throwable;
 
-  /// A Throwable decorator that holds a Mechanism related to the decorated Throwable
+  /// A throwable decorator that holds a [Mechanism] related to the decorated
+  /// [throwable]
   ///
-  /// Use the 'throwable' directly if you don't want the decorated Throwable
+  /// Use the [throwable] directly if you don't want the decorated throwable
   dynamic get throwableMechanism => _throwable;
 
-  /// an optional attached StackTrace
-  /// used when event has no throwable or exception, see [SentryOptions.attachStacktrace]
-  final SentryStackTrace? stackTrace;
+  /// One or multiple chained (nested) exceptions that occurred in a program.
+  final List<SentryException>? exceptions;
 
-  /// an exception or error that occurred in a program
-  /// TODO more doc
-  final SentryException? exception;
+  /// The Threads Interface specifies threads that were running at the time an
+  /// event happened. These threads can also contain stack traces.
+  /// Typically not needed in Dart applications.
+  final List<SentryThread>? threads;
```

Comment:
as Dart is single-threaded or errors are not shared across multiple Isolates, maybe we don't add it? that was the reason in the 1st place

Reply:
I thought that we could add the Java thread (or whatever the platforms language is) when adding mixed stacktraces. Though I don't know if that would work.

Reply:
nvm, its useful because sometimes we add the threads interface because of `Current Isolate`

Reply:
We could replace `Current Isolate` with [Isolate.debugName](https://api.flutter.dev/flutter/dart-isolate/Isolate/debugName.html) if available.

Reply:
yeah I guess I checked that before, iirc